src/game/boe.main.cpp

```diff
@@ -135,21 +149,64 @@ int main(int argc, char* argv[]) {
 	}
 }
 
-static void init_sbar(std::shared_ptr<cScrollbar>& sbar, rectangle rect, int max, int pgSz, int start = 0) {
+static void init_sbar(std::shared_ptr<cScrollbar>& sbar, std::string const name, rectangle rect, rectangle events_rect, int max, int pgSz, int start = 0) {
 	sbar.reset(new cScrollbar(mainPtr));
 	sbar->setBounds(rect);
 	sbar->setMaximum(max);
 	sbar->setPosition(start);
 	sbar->setPageSize(pgSz);
+	sbar->set_mousewheel_event_catching_rect(events_rect);
 	sbar->hide();
+	
+	drawable_mgr.add_drawable(UI_LAYER_DEFAULT, name, sbar);
+	event_listeners[name] = std::dynamic_pointer_cast <iEventListener> (sbar);
+}
+
+static void init_scrollbars () {
+
+	// Cover entire message log + scrollbar
+	rectangle const message_log_events_rect {
+		win_to_rects[WINRECT_TRANSCRIPT].top,
+		win_to_rects[WINRECT_TRANSCRIPT].left,
+		sbar_rect.bottom,
+		sbar_rect.right
+	};
+	
+	// Cover entire inventory + scrollbar
+	rectangle const inventory_events_rect {
+		win_to_rects[WINRECT_INVEN].top,
+		win_to_rects[WINRECT_INVEN].left,
+		item_sbar_rect.bottom,
+		item_sbar_rect.right
+	};
+	
+	// MAGIC NUMBERS: max size, step size, initial position - all in abstract "step" units
+	init_sbar(text_sbar, "message-log-scrollbar", sbar_rect, message_log_events_rect, 58, 11, 58);
```

Comment:
I realize this is a silly little thing, but can we perhaps call this `"transcript-scrollbar"`?

Reply:
done